src/hal/drivers/mesa-hostmot2/bspi.c

```diff
@@ -197,9 +197,21 @@ int hm2_bspi_write_chan(char* name, int chan, rtapi_u32 val)
     return r;
 }
 
+int hm2_bspi_clear_counter(hostmot2_t *hm2, int i)
+{
+    rtapi_u32 buff = 0; // data is dont care when writing to count register
+    int r;
+    r = hm2->llio->write(hm2->llio, hm2->bspi.instance[i].count_addr, &buff, sizeof(rtapi_u32));
+    if (r < 0) {
+        HM2_ERR("BSPI: hm2->llio->write failure %s\n", hm2->bspi.instance[i].name);
+    }
+
+    return r;
+}
+
```

Comment:
This added function appears to be unused.  If that is correct, can you please remove it?